packages/yoga/src/Popover/web/Popover.jsx

```diff
@@ -25,6 +41,10 @@ function Popover({
     setIsPopoverOpen(false);
   };
 
+  const toggleOpenPopover = () => {
+    setIsPopoverOpen(!isPopoverOpen);
+  };
```

Comment:
This is a bad practise. Use[ setState callback](https://carlogino.com/blog/react-setstate-callback-function) :)

```suggestion
  const toggleOpenPopover = () => {
      setIsPopoverOpen(current => !current);
  };
```